.github/release-template.md

```diff
@@ -0,0 +1,18 @@
+This week's push hero is @{{GITHUB_USER}}
```

Comment:
Shouldn't this be an input too?  The current push hero (who creates this doc) is very rarely also the person doing it next time.

Reply:
That's a good point. Kind of unfortunate though because you could end up doing it wrong... I actually think you can use select inputs in workflow dispatch so maybe that would be better. Hardcode the github handles and then you just select the right one.

Reply:
Yeah.  A select on our 3 github handles would be ideal, but being able to manually edit to choose the correct one is fine.

Reply:
Updated with the select.